src/main/java/Entities/Playlist.java

```diff
@@ -1,10 +1,10 @@
 package Entities;
 
-import java.util.ArrayList;
 import java.util.LinkedList;
 
 /**
  * Entities layer class representing a Playlist.
+ * TODO Edge Case: song is removed while its being played should not affect the songs being played, or playlists
```

Comment:
Nice catch, will need to work out the kinks with this later when the front-end stuff is more developed.